src/main/java/InvalidListIndexException.java

```diff
@@ -0,0 +1,15 @@
+public class InvalidListIndexException extends DukeException {
+
+    /**
+     * Gives the String representation of the Exception
+     *
+     * @return a String
+     */
+    public String toString() {
+        return "______________________________________________________"
+                + "\n"
+                + " OOPS!!! The index you've chosen is invalid"
+                + "\n"
+                + "______________________________________________________";
```

Comment:
I like that you have specific exceptions for a possible error the user might run into 👍 

Would it be better to extract the templating part into a helper function such that you can change it once for all exceptions if needed?